security.md

```diff
@@ -0,0 +1,59 @@
+# Security policy
+
+We believe in transparency to mitigate security risks. All known vulnerabilities are available on 
+our [security page](docs/modules/ROOT/pages/security.adoc).

```

Comment:
404

Reply:
@hoangdat can you help me with this ? not sure about vulnerabilities